### ISSUE TITLE
Styling bug on Address component

### ISSUE THREAD

**Issue by adamhaeger** [Altinn/app-frontend-react#2520]
Discussion: https://altinn.slack.com/archives/C02EJ9HKQA3/p1727169859566029

Recreate:

1. Add an Address component to an app, make required.
2. Trigger validation by submtting the form

Expected:

Styling according to design

Actual:

![Image](https://github.com/user-attachments/assets/db963dd4-3890-4f96-9f3c-866e7ff261f3)

**Comment by rodsjo** [comment]
Ønsker at denne blir fikset etter hvert. Vi bruker Address-komponenten i et skjema som skal produksjonssettes i slutten av november
